core/frontend/src/tile/map/ImageryProviders/ArcGISMapLayerImageryProvider.ts

```diff
@@ -283,7 +283,7 @@ export class ArcGISMapLayerImageryProvider extends MapLayerImageryProvider {
           });
         if (token?.token)
           tokenParam = `&token=${token.token}`;
-      } catch {
+      } catch (_) {
```

Comment:
Why do so many otherwise untouched files have this kind of change?

Reply:
I was seeing certa test failures in the instrumented test without this change, not entirely sure why. Complained about 
[`744d27a` (#2206)](https://github.com/imodeljs/imodeljs/pull/2206/commits/744d27a94df3cf211249edfb4c839d63873d6b02)

```
ERROR in ./lib/cjs/tile/map/ImageryProviders/ArcGISMapLayerImageryProvider.js
    Module build failed (from C:/Users/ArunG/Repos/GH/imjs/imodeljs/common/temp/node_modules/.pnpm/istanbul-instrumenter-loader@3.0.1_webpack@4.42.0/node_modules/istanbul-instrumenter-loader/dist/cjs.js):
    SyntaxError: Unexpected token, expected ( (252:18)
```

Reply:
I wonder if this was caused by updating the target?  Maybe the istanbul-instrumenter-loader's parser doesn't recognize [optional catch bindings](https://github.com/tc39/proposal-optional-catch-binding), which are new in ES2019?

Reply:
Actually, @aruniverse - is this an issue w/ webpack's parser instead?  You commented out the instrumented build, did you still hit these errors after that?

Reply:
Just looked into this some more, and that error is definitely coming from istanbul-instrumenter-loader.  Let's just revert all these `catch (_)` changes, disable all the certa coverage/istanbul stuff for now, and follow up w/ another PR that fixes coverage for those tests.

Reply:
Done.